droidlet/tools/artifact_scripts/upload_artifacts_to_aws.py

```diff
@@ -14,6 +14,72 @@
 print("Rootdir : %r" % ROOTDIR)
 
 
+def tar_and_upload(agent, artifact_name, model_name=None):
```

Comment:
I don't understand why we need this new function ? The function : `compute_checksum_tar_and_upload` does exactly this right ?